app/lib/dartdoc/models.dart

```diff
@@ -16,23 +16,58 @@ import 'storage_path.dart' as storage_path;
 
 part 'models.g.dart';
 
+/// Describes the details of a dartdoc-generated content.
 @JsonSerializable()
 class DartdocEntry {
+  /// Unique identifier of the entry, generated via a random UUID.
```

Comment:
Could we instead write:
"Random uuid for lookup in storage bucket, see storage_paths"

I can guess what a uuid is, and that it's random (well, one can hope it's random), but what is it good for :) hehe